### DISCUSSION TITLE
Are there any fees for using Planetary Computer

### DISCUSSION THREAD

**Discussion by slesinskip** [microsoft/PlanetaryComputer#204] [General]
I have a question about possible fees for using the PC services. I know that the access to the data is free of charge but i have additional questions.

1. Are data processing on Planetary Computer Hub with computing environment may incur some costs? What are the limitations?
2. What is the amount of basic computing environment storage and whether there may be some costs involved for using this storage. 

Thank you in advance for help.

**Comment by TomAugspurger** [comment]
The [Planetary Computer Hub](https://planetarycomputer.microsoft.com/docs/overview/environment/) is just a [JupyterHub](https://jupyter.org/hub) deployment. There's no cost to you for using it. It has a small home directory for storing [code and small outputs](https://planetarycomputer.microsoft.com/docs/overview/environment/#understanding-the-file-system). For larger outputs, we recommend using Azure Blob Storage in your own subscription, which would incur a cost against your Azure subscription.
